app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
`||=` có tác dụng gì

Reply:
theo em thì ||= giúp giảm việc thực hiện phép toán ở vế phải, chỉ thực hiện lại khi vế trái nil hoặc false ạ

Reply:
cụ thể trong trường hợp này thì nó hoạt động ntn em?

Reply:
||= trong trường hợp này thì việc truy xuất thông tin và gán vào current_user khi current user nil hay false thôi ạ